packaging.mk

```diff
@@ -29,21 +29,31 @@ DOCKER_IMAGES := \
 	build/docker/apm-server-$(APM_SERVER_VERSION).txt \
 	build/docker/apm-server-$(APM_SERVER_VERSION)-SNAPSHOT.txt \
 	build/docker/apm-server-ubi-$(APM_SERVER_VERSION).txt \
-	build/docker/apm-server-ubi-$(APM_SERVER_VERSION)-SNAPSHOT.txt
+	build/docker/apm-server-ubi-$(APM_SERVER_VERSION)-SNAPSHOT.txt \
+	build/docker/apm-server-wolfi-$(APM_SERVER_VERSION).txt \
+	build/docker/apm-server-wolfi-$(APM_SERVER_VERSION)-SNAPSHOT.txt
 
 build/docker/%.txt: DOCKER_IMAGE_TAG := docker.elastic.co/apm/apm-server:%
 build/docker/%.txt: VERSION := $(APM_SERVER_VERSION)
 build/docker/%-SNAPSHOT.txt: VERSION := $(APM_SERVER_VERSION)-SNAPSHOT
 build/docker/apm-server-ubi-%.txt: DOCKER_BUILD_ARGS+=--build-arg BASE_IMAGE=docker.elastic.co/ubi9/ubi-minimal
+build/docker/apm-server-wolfi-%.txt: DOCKER_BUILD_ARGS+=--build-arg BASE_IMAGE=docker.elastic.co/wolfi/chainguard-base:20230214 --build-arg BASE_GOLANG_IMAGE=docker.elastic.co/wolfi/go
+
+INTERNAL_DOCKER_IMAGE := docker.elastic.co/observability-ci/apm-server-internal
```

Comment:
This should help create docker images and publish them to our internal docker registry - this is handy for validating changes before anything related to the DRA and in GitHub actions.

We are safe since these docker images won't be public available for the time being:

```bash
$ docker pull docker.elastic.co/observability-ci/apm-server-internal:8.15.0-SNAPSHOT-wolfi
Error response from daemon: pull access denied for docker.elastic.co/observability-ci/apm-server-internal, repository does not exist or may require 'docker login': denied: requested access to the resource is denied
```